WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/download/BackupDownloadViewModel.kt

```diff
@@ -143,6 +154,12 @@ class BackupDownloadViewModel @Inject constructor(
         _toolbarStateObservable.value = toolbarState
     }
 
+    fun transitionToError() {
+        backupDownloadState = backupDownloadState.copy(isError = true)
+        wizardManager.setCurrentStepIndex(BackupDownloadStep.indexForErrorTransition())
+        wizardManager.showNextStep()
+    }
```

Comment:
Can we add a unit test for this scenario when (given in progress step) wizard invokes next step on error?

Reply:
I'm going to add tests in a separate task, as I have a few others to add for the new error types now